### ISSUE TITLE
Add numLines in genBlob (org title: Make random deterministic from a provided seed)

### ISSUE THREAD

**Issue by bonafidehan** [artt/react-random-shapes#3]
It'd be a nice effect to build line after line if you increase the number of lines. Would you be able to expose an option that sets the seed for the randomness? Thanks!

**Comment by artt** [comment]
Hey, thanks for your interest! I'm not quite sure I understand what you mean. Could you elaborate a bit more? Maybe your use case? Thanks!

**Comment by bonafidehan** [comment]
Thanks for the quick response! So you're using Math.random which is a PRNG that can't take a seed that controls its sequence of random numbers. Many PRNG libraries take a seed that makes it deterministic.

My use case has 2 parts: 1/ I want to overlay multiple blobs on top of each other 2/ I want to progressively add more blobs where the previous blobs are stable and don't change. Here's an example of the end product: [
![Screenshot from 2021-01-28 20-49-54](https://user-images.githubusercontent.com/536523/106233086-8a721900-61aa-11eb-8f7b-ae45d9b82e7f.png)
](url)

**Comment by artt** [comment]
Got it. Seems like a quick fix. Won't have time until this weekend though, so if you'd like, please feel free to submit a PR :)

**Comment by bonafidehan** [comment]
Awesome! I'll add it to my todo list but very happy to wait until wknd when you get to it. Would super appreciate it!

**Comment by artt** [comment]
OK I published 0.5.0 and that should work. Example:

```js
<RandomHLine width={600} height={300} options={{classNameTop: "abc", seed: "hello!"}} />
```

Let me know if you got it working for you. Thanks for the idea!
Gonna close this for now. Feel free to lmk if you spot any problem.

**Comment by bonafidehan** [comment]
Thanks so much! How about the overlay of multiple blobs on top of each other? I can do that manually outside of the library. But just like RandomHLine, if there was a numBlobs for RandomBlob it would make the library more powerful.

**Comment by artt** [comment]
Hm that could be coming up... Just to confirm... You don't need per-blob seed right?

**Comment by bonafidehan** [comment]
Nope! Don't need per-blob seed. I need a seed for the entire N-blob structure. That way, if I had a 3-blob diagram, and then I wanted a 4-blob diagram, it would generate one more blob and the 3 previous blobs would remain the same. That would work only if the seed is for the entire structure. Example of a 3-blob structure:
![106233086-8a721900-61aa-11eb-8f7b-ae45d9b82e7f](https://user-images.githubusercontent.com/536523/106375063-afe35c00-633d-11eb-85d1-6393e9cb3944.png)

**Comment by artt** [comment]
Thanks! Not sure when I'll have time to do this... but again please feel free to submit a PR. Need to make a change in the [core package](https://github.com/artt/random-shapes) as well as this one.

**Comment by bonafidehan** [comment]
Created a PR, but wasn't sure how to test. What's your test flow? Thanks! https://github.com/artt/random-shapes/pull/1

**Comment by artt** [comment]
Sorry it took me a while. Just reviewed. I'm pretty new to all this, so I'm not sure how to create a test yet. If you know of a tutorial/doc I could read on making tests for packages, please lmk.

**Comment by bonafidehan** [comment]
Just got back to this. Thanks for the review! No suggestion from my side, but just knowing that I didn't break stuff would be great :) Not sure how you did that on your side.

**Comment by artt** [comment]
Just guesses :P

I just merged your PR and published `random-shapes@0.2.1`. Please feel free to submit a PR for this package if you have time. Thanks so much!

**Comment by bonafidehan** [comment]
Created https://github.com/artt/react-random-shapes/pull/4. Let me know if there's anything I need to change!

**Comment by bonafidehan** [comment]
Thanks for the suggestions, fixed!

**Comment by bonafidehan** [comment]
Hey @artt! Any update on that PR review?

**Comment by artt** [comment]
Sorry. I was looking for your PR the other day but must have looked at the core repo.

Merged, and I updated readme + example as well. Published as [0.6.0](https://www.npmjs.com/package/react-random-shapes).

Thanks again!
